src/site/_includes/components/header/macro.njk

```diff
@@ -0,0 +1,59 @@
+{#
+ This macro is based on the GOV.UK equivalent from the GOV.UK Frontend:
+ https://github.com/alphagov/govuk-frontend/blob/cdfc281bebf0afb6ea121ee8f52eb332dfc40615/src/govuk/components/header/template.njk
+
+ The logo has been changed, some CSS classes have been added and there are some slight markup changes.
+#}
+
+{% macro header(params) %}
```

Comment:
The macros I added have a guidance prefix e.g. `guidanceHeader` – would be good to make it consistent.